src/pages/Pools/PairsByToken0/index.tsx

```diff
@@ -0,0 +1,201 @@
+                      Pairs
+                    </TYPE.mediumHeader>
+                  </UndecoratedLink>
+                </Box>
+                <Box mr="8px">
+                  <TYPE.mediumHeader fontWeight="400" fontSize="26px" lineHeight="32px" color="text4">
+                    /
+                  </TYPE.mediumHeader>
+                </Box>
+                <PointableFlex onClick={handleAllClick}>
+                  <Box mr="4px">
+                    <CurrencyLogo currency={token0 || undefined} size="20px" />
+                  </Box>
+                  <Box mr="4px">
+                    <Text fontWeight="600" fontSize="16px" lineHeight="20px">
+                      {token0?.symbol}
+                    </Text>
+                  </Box>
+                </PointableFlex>
+                <Box>
+                  <ChevronDown size={12} />
+                </Box>
+              </Flex>
+              <ButtonRow>
+                <ResponsiveButtonPrimary id="join-pool-button" as={Link} padding="8px 14px" to="/create">
+                  <Text fontWeight={700} fontSize={12}>
+                    CREATE PAIR
+                  </Text>
+                </ResponsiveButtonPrimary>
+                {liquidityMiningEnabled && (
+                  <ResponsiveButtonSecondary as={Link} padding="8px 14px" to="/liquidity-mining/create">
+                    <Text fontWeight={700} fontSize={12} lineHeight="15px">
+                      CREATE LIQ. MINING
+                    </Text>
+                  </ResponsiveButtonSecondary>
+                )}
+              </ButtonRow>
+            </TitleRow>
+            <PairsList loading={loadingPairs} pairs={pairs} />
+          </AutoColumn>
+        </AutoColumn>
+        {/* TODO: uncomment once multi-chain analytics are a thing */}
+        {/* <ButtonWithLink
+          link={`https://dxstats.eth.link/#/account/${account}`}
+          text={'ACCOUNT ANALYTICS AND ACCRUED FEES'}
+          marginTop="32px"
+        /> */}
+        {/* Should not be needed since when we fetch liquidity positions from the subgraph */}
+        {/* <TYPE.body color="text4" textAlign="center" fontWeight="500" fontSize="14px" lineHeight="17px" marginTop="32px">
+          Don't see a pool you joined?{' '}
+          <StyledInternalLink color="text5" id="import-pool-link" to="/find">
+            Import it.
+          </StyledInternalLink>
+        </TYPE.body> */}
+
+        <VoteCard style={{ marginTop: '32px' }}>
+          <CardSection>
+            <AutoColumn gap="md">
+              <RowBetween>
+                <TYPE.body fontWeight={600} lineHeight="20px">
+                  Liquidity provider rewards
+                </TYPE.body>
+              </RowBetween>
+              <RowBetween>
+                <TYPE.body fontWeight="500" fontSize="12px" lineHeight="20px" letterSpacing="-0.4px">
+                  Liquidity providers earn a swap fee (0.25% by default) on all trades proportional to their share of
```

Comment:
This isn't quite right because the protocol fee takes part of the swap fee

Reply:
Right, I've changed the text here c115d1d8ad45bdc42ded15c1f057b2dc3a1e77e6, would love some feedback